### ISSUE TITLE
Focal Loss - do we have duplicate?

### ISSUE THREAD

**Issue by innat** [keras-team/keras-cv#1117]
- [keras_cv.losses.FocalLoss](https://github.com/keras-team/keras-cv/blob/master/keras_cv/losses/focal.py)
- [tf.keras.losses.BinaryFocalCrossentropy](https://www.tensorflow.org/api_docs/python/tf/keras/losses/BinaryFocalCrossentropy)

**Comment by DavidLandup0** [comment]
Not a duplicate. The KCV one is a self-adjusting focal loss that switches between L1 and L2, while the Keras one is crossentropy imbued with focal weighing for classes (so BCE + Focal)

**Comment by cybergeekgyan** [comment]
I would like to work on this issue can you assign this to me @innat

**Comment by tanzhenyu** [comment]
> I would like to work on this issue can you assign this to me @innat

Done -- Would you mind helping us evaluate whether we could directly move to keras loss instead, by comparing the before / after model quality

**Comment by innat** [comment]
@tanzhenyu 
> evaluate whether we could directly move to keras loss instead, by comparing the before / after model quality

Instead it would be sufficient just to compare the implementation details. Fortunately, they're easy to read.
